modules/views/components/civicrm.core.inc

```diff
@@ -2816,6 +2863,23 @@ function _civicrm_core_data(&$data, $enabled) {
       'handler' => 'views_handler_sort',
     ),
   );
+
+  $data['civicrm_group']['title_'] = array(
+    'title' => t('Title'),
+    'help' => t('Group Title'),
+    'real field' => 'title',
+    'field' => array(
+      'handler' => 'views_handler_field',
+      'click sortable' => TRUE,
+    ),
+    'argument' => array(
+      'handler' => 'views_handler_argument',
+    ),
+    'sort' => array(
+      'handler' => 'views_handler_sort',
+    ),
+  );
+
```

Comment:
The new title is not included in view filters. Was this intentional? Though it contains the deprecated title.

Reply:
Not sure why but have updated it now @jackrabbithanna